packages/components/src/Tag/subcomponents/LiveIcon/LiveIcon.tsx

```diff
@@ -0,0 +1,67 @@
+// This file is autogenerated by wrapSVGs.ts
+// Changes to this file will be overwritten
+
+import React from 'react'
+import { SVG } from '~components/Icon/subcomponents/SVG'
+import type { IconProps } from '~components/Icon/types'
+import styles from './LiveIcon.module.css'
+
+export const LiveIcon = (): JSX.Element => (
+  <span className={styles.liveIcon}>
+    <LiveIconContent
+      role="presentation"
+      classNameOverride={styles.liveIcon_base}
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+    />
+    <LiveIconContent
+      role="presentation"
+      classNameOverride={styles.liveIcon_1}
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+    />
+    <LiveIconContent
+      role="presentation"
+      classNameOverride={styles.liveIcon_2}
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+    />
+    <LiveIconContent
+      role="presentation"
+      classNameOverride={styles.liveIcon_3}
+      width="16"
+      height="16"
+      viewBox="0 0 16 16"
+      fill="none"
+    />
+  </span>
+)
+
+const LiveIconContent = (props: IconProps): JSX.Element => {
+  const svgContent = (
+    <>
+      <path
+        stroke="currentColor"
+        d="M3.266 12.733c-2.6-2.6-2.6-6.866 0-9.466M5.2 10.8c-1.534-1.533-1.534-4.067 0-5.667"
+        strokeWidth="1.5"
+        strokeLinejoin="round"
+        strokeLinecap="round"
+      />
+      <path fill="currentColor" d="M8 10a2 2 0 1 0 0-4 2 2 0 0 0 0 4" />
+      <path
+        stroke="currentColor"
+        d="M10.8 5.2c1.533 1.533 1.533 4.067 0 5.667M12.733 3.267c2.6 2.6 2.6 6.8 0 9.4"
+        strokeWidth="1.5"
+        strokeLinejoin="round"
+        strokeLinecap="round"
+      />
+    </>
+  )
+  return <SVG {...props}>{svgContent}</SVG>
+}
```

Comment:
Given it's a subcomponent that we will not be exposing, we can just be very deliberate with our composition :)

```suggestion
const LiveIconSvg = (props: IconProps): JSX.Element => (
  <SVG 
    // add the attributes that do not differ between your usages eg. role
    {...props}
  >
    <path
      stroke="currentColor"
      d="M3.266 12.733c-2.6-2.6-2.6-6.866 0-9.466M5.2 10.8c-1.534-1.533-1.534-4.067 0-5.667"
      strokeWidth="1.5"
      strokeLinejoin="round"
      strokeLinecap="round"
    />
    <path fill="currentColor" d="M8 10a2 2 0 1 0 0-4 2 2 0 0 0 0 4" />
    <path
      stroke="currentColor"
      d="M10.8 5.2c1.533 1.533 1.533 4.067 0 5.667M12.733 3.267c2.6 2.6 2.6 6.8 0 9.4"
      strokeWidth="1.5"
      strokeLinejoin="round"
      strokeLinecap="round"
    />
  </SVG>
)
```

Reply:
Nit: I'd also suggest that given the reading flow, it is easier when the subcomponent is put above (if not in another file) the consuming component/function.